src/components/bottomDrawer/bottomDrawer.css

```diff
@@ -35,6 +35,8 @@
 }
 
 .content-wrapper {
```

Comment:
`content-wrapper` for non custom info pins has funky styling
<img width="1234" alt="screen shot 2018-07-11 at 10 41 38 am" src="https://user-images.githubusercontent.com/1947857/42589898-42034244-84f7-11e8-888a-6eeedc4f73f2.png">